lib/install/config/assets.js

```diff
@@ -0,0 +1,40 @@
+// Handles static asset integration - like sass and images
+// Note: You must restart bin/webpack-watcher for changes to take effect
+
+const ExtractTextPlugin = require('extract-text-webpack-plugin')
+const process = require('process')
+const sharedConfig = require('./shared.js')
+const { devServer } = require('../../package.json')
+
+const production = process.env.NODE_ENV === 'production'
+const hotServerAddr = `http://${devServer.host}:${devServer.port}/`
```

Comment:
Not if it is passed in from an environment variable and that environment variable is read from `config.x.webpacker[:dev_server_host]`. I think I might need to take your branch and make the changes I'm proposing to make things clear.